### PR TITLE
Add Specs: Cosmos hub

### PR DESCRIPTION
**Pull request by tecnodes-network**
Name: Tecnodes
Discord: tecnodes
Commission : 3%

At Tecnodes, we pride ourselves on being at the forefront of blockchain technology, dedicated to contributing to the growth and decentralization of blockchain networks. Our journey in this space has been marked by a commitment to innovation, security, and active participation in the most cutting-edge blockchain projects. As a company, we specialize in providing validation services across multiple blockchain networks, ensuring their robustness, security, and continued operation. Our team comprises seasoned professionals with a deep understanding of both Web2 and Web3 technologies, making us a versatile and reliable partner in the blockchain ecosystem. One of our most exciting adventure has been with the Lava Network, where we have been early supporters since the inception of their testnet. Lava, a highly innovative blockchain project, has always been at the cutting edge of technological advancements, and we are thrilled to have been a part of its journey from the very beginning. Our involvement with Lava spans across all their testnets, where we have consistently served as both a validator and a provider.  Currently, we are operating on the Lava Mainnet as both a validator and a provider. This role is not just a continuation of our commitment but a testament to our capabilities in maintaining and securing blockchain networks at the highest level.  We are thrilled to be stepping into the role of Spec Champion for Lava Mainnet. As longstanding supporters of the Lava ecosystem, having participated as validators and providers across all testnets and mainnet, this opportunity to contribute further is incredibly exciting. In this role, we’ll be responsible for crafting and maintaining the specifications for new chains on Lava, ensuring they are up to date with the latest APIs and chain updates. This not only supports the Lava governance process but also helps guide and assist RPC providers. We look forward to working closely with the Lava community and governance to add new chains and support the network’s growth.

Tecnodes' involvement with Lava is a reflection of our broader mission: to support and strengthen the blockchain networks that form the backbone of the decentralized future. We believe that by providing top-tier validation services and participating in key roles like Specs Champion, we are helping to build a more secure, transparent, and decentralized world. Our work with Lava is just one example of how we are making a difference in the blockchain space, and we are eager to continue this journey with other forward-thinking projects in the industry.

### CODE REVIEW

File: mainnet-1/specs/cosmoshub.json [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+        "title": "Add Specs: Cosmos hub",
+        "description": "Adding new specification support for relaying cosmoshub data on Lava Mainnet",
+        "specs": [
+            {
+                "index": "COSMOSHUB",
+                "name": "cosmos hub mainnet",
+                "enabled": true,
+                "imports": [
+                    "COSMOSSDKFULL",
+                    "COSMOSSDK45DEP"
```
Comment (omerlavanet) [inline mainnet-1/specs/cosmoshub.json:12]: this inheritance is outdated as cosmos hub is already on v50

File: mainnet-1/specs/cosmoshub.json [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+                ],
+                "reliability_threshold": 268435455,
+                "data_reliability_enabled": true,
```
Comment (omerlavanet) [inline mainnet-1/specs/cosmoshub.json:15]: missing contributor field

File: mainnet-1/specs/cosmoshub.json [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+                "block_distance_for_finalized_data": 0,
+                "blocks_in_finalization_proof": 1,
+                "average_block_time": 6500,
+                "allowed_block_lag_for_qos_sync": 2,
+                "shares": 1,
+                "min_stake_provider": {
+                    "denom": "ulava",
+                    "amount": "100000000"
+                },
+                "api_collections": [
```
Comment (omerlavanet) [inline mainnet-1/specs/cosmoshub.json:25]: apis are not updated with the full list of cosmos 50 for example interchain queries

File: mainnet-1/specs/cosmoshub.json (hunk 1 above)
Comment (tecnodes-network) [reply mainnet-1/specs/cosmoshub.json:12]: OK

File: mainnet-1/specs/cosmoshub.json (hunk 2 above)
Comment (tecnodes-network) [reply mainnet-1/specs/cosmoshub.json:15]: ok

File: mainnet-1/specs/cosmoshub.json (hunk 3 above)
Comment (tecnodes-network) [reply mainnet-1/specs/cosmoshub.json:25]: Ok

**Review by omerlavanet — CHANGES_REQUESTED** [review]

**Pull request closed without merging**